src/firefox-infobar-ui/static/_locales/ru/messages.json

```diff
@@ -0,0 +1,50 @@
+{
+  "extensionName": {
+    "message": "Firefox Переводчик",
+    "description": "Name of the extension"
+  },
+  "extensionDescription": {
+    "message": "Нейронный машинный перевод для браузера.",
+    "description": "Description of the extension."
+  },
+  "currentlyDownloadingTranslationEngine": {
+    "message": "Скачивание движка перевода",
+    "description": "Informs the user that the translations feature is currently downloading the translation engine."
+  },
+  "currentlyDownloadingLanguageModel": {
+    "message": "Скачивание языковой модели",
+    "description": "Informs the user that the translations feature is currently downloading the language model files."
+  },
+  "detailedDownloadProgress": {
+    "message": "$PERCENTAGE$% из $MB$ МБ загружено",
+    "description": "Informs the user how the language model file downloading is progressing.",
+    "placeholders": {
+      "percentage": {
+        "content": "$1",
+        "example": "50"
+      },
+      "mb": {
+        "content": "$2",
+        "example": "22.2"
+      }
+    }
+  },
+  "currentlyLoadingLanguageModel": {
+    "message": "Загрузка языковой модели",
+    "description": "Informs the user that the translations feature is currently loading the language model files into memory."
+  },
+  "loadedLanguageModel": {
+    "message": "Языковая модель загружена",
+    "description": "Informs the user that the translations feature is has loaded the language model files into memory."
```

Comment:
**is** should be removed from the _description_ statement. I guess it exists in other files as well.

Reply:
fixed